files/templates/mbta/email/messages/messages_en.properties

```diff
@@ -1,3 +1,15 @@
-secondFactor.emailSubject=Code verification
+secondFactor.emailSubject=MBTA verification code
 secondFactor.emailBody=Your verification code is: {0}
 secondFactor.emailBodyHtml=Your verification code is: {0}
+
+executeActionsSubject=Verify your email
+executeActionsBody=Someone has created an {2} account with this email address. If this was you, click the link below to verify your email address.\n\n{0}\n\nThis link will expire within {4}.\n\nIf you didn't create this account, ignore this message.
+executeActionsBodyHtml=<p>Someone has created an {2} account with this email address. If this was you, click the link below to verify your email address.</p><p><a href="{0}">Verify your email</a></p><p>This link will expire within {4}.</p><p>If you didn''t create this account, ignore this message.</p>
+
+passwordResetSubject=Reset your MBTA password
+passwordResetBody=Someone just requested a reset of your {2} account password. If this was you, please click the link below to reset your password.\n\n{0}\n\nThis link and code will expire within {3}.\n\nIf you don't want to reset your password, ignore this message.
+passwordResetBodyHtml=<p>Someone just requested a reset of your {2} account password. If this was you, please click the link below to reset your password.</p><p><a href="{0}">Reset my MBTA account password</a></p><p>This link will expire within {3}.</p><p>If you don''t want to reset your password, ignore this message.</p>
+
+emailVerificationSubject=Verify your email
+emailVerificationBody=Someone has created an {2} account with this email address. If this was you, click the link below to verify your email address.\n\n{0}\n\nThis link will expire within {3}.\n\nIf you didn't create this account, ignore this message.
+emailVerificationBodyHtml=<p>Someone has created an {2} account with this email address. If this was you, click the link below to verify your email address.</p><p><a href="{0}">Verify your email</a></p><p>This link will expire within {3}.</p><p>If you didn''t create this account, ignore this message.</p>
```

Comment:
Nitpick: Missing newline at end of file (this is another thing you should be able to configure your editor to add automatically on save)